src/mon/MonCommands.h

```diff
@@ -712,6 +712,13 @@ COMMAND("osd create " \
 	"name=uuid,type=CephUUID,req=false " \
 	"name=id,type=CephOsdName,req=false", \
 	"create new osd (with optional UUID and ID)", "osd", "rw", "cli,rest")
+COMMAND("osd new " \
+        "name=replace_id,type=CephOsdName,req=true " \
```

Comment:
if we do make `osd new` also create a fresh osd (which makes a lot more sense, given the command name), then we should also make `uuid` optional and basically deduplicate the code from `osd create`.